layouts/partials/page-meta-links.html

```diff
@@ -8,7 +9,11 @@
 {{ end }}
 {{ $issuesURL := printf "%s/issues/new?title=%s" $gh_repo (htmlEscape $.Title )}}
 <a href="{{ $editURL }}" target="_blank"><i class="fa fa-edit fa-fw"></i> {{ T "post_edit_this" }}</a>
-<a href="{{ $issuesURL }}" target="_blank"><i class="fab fa-github fa-fw"></i> {{ T "post_create_issue" }}</a>
+<a href="{{ $issuesURL }}" target="_blank"><i class="fas fa-book fa-fw"></i> {{ T "post_create_issue" }}</a>
```

Comment:
I do like the options for project vs docs issues but I wonder if we are losing context without the GitHub icon (ie. that the link opens to GitHub)? 

I think we could get away with just the GitHub icon given that the first link refers to this page, so one could assume that "Create Issue" also refers to this page?

Im dont feel to strongly about this either way but the thought crossed my mind.

Reply:
I've reverted to the GitHub icon for the "Create documentation link". I also don't feel strongly about it either, as long as there is some visual distinction between the icons and it's easy to override if you prefer something else.